include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -2354,6 +2354,68 @@ vertex_dof_index (const unsigned int vertex,
 
 template <template <int, int> class DoFHandlerType, int spacedim, bool level_dof_access>
 inline
+types::global_dof_index
+DoFAccessor<0,DoFHandlerType<1,spacedim>, level_dof_access>::
+dof_index (const unsigned int i,
+           const unsigned int fe_index) const
+{
+  AssertThrow(false, ExcNotImplemented());
```

Comment:
Why use `AssertThrow` here and below? We've typically just used `Assert` for functions that are not implemented, which immediately terminates the program and makes it much easier to find the place than if you had to set a breakpoint on throw statements.